src/veh_interact.cpp

```diff
@@ -38,12 +38,18 @@
 #include <numeric>
 #include <string>
 
-static inline const char *status_color( bool status )
+static inline const std::string status_color( bool status )
 {
-    static const char *good = "green";
-    static const char *bad = "red";
+    static const std::string good = "<color_green>";
```

Comment:
Allocating them statically is nice and all, but because this returns a *value* (and a const one again) the string will be copied again anyway.

May I suggest the simple and short alternative:
```C++
return status ? "color_green" : "color_red";
```
(Only requires one string construction per function call.)

Reply:
Thanks!